lib/gemfury/command/app.rb

```diff
@@ -249,7 +249,7 @@ def git_config_set(repo, *vars)
     end
   end
 
-  desc "git:config:unset", "Remove variables from Git repository's build environment"
+  desc "git:config:unset NAME KEYS", "Remove variables from Git repository's build environment. KEYS: KEY1 KEY2"
```

Comment:
Same as above, for simplicity:

```suggestion
  desc "git:config:unset REPO_NAME KEY...", "Remove variables from Git repository's build environment."
```